ui/pages/home/home.component.stories.tsx

```diff
@@ -0,0 +1,350 @@
+import React from 'react';
```

Comment:
Adding story for future iterations of hard to reach UI code



https://github.com/user-attachments/assets/d6a9abaf-9746-4dc2-b180-96bba2b863f4